src/main/java/gov/cabinetoffice/gap/adminbackend/services/ApplicationFormService.java

```diff
@@ -44,12 +48,33 @@ public class ApplicationFormService {
 
     private final TemplateApplicationFormRepository templateApplicationFormRepository;
 
+    private final SchemeRepository schemeRepository;
+
     private final ApplicationFormMapper applicationFormMapper;
 
     private final SessionsService sessionsService;
 
     private final Validator validator;
 
+    private final Clock clock;
+
+    public ApplicationFormEntity save(ApplicationFormEntity applicationForm) {
+        final ApplicationFormEntity savedApplicationForm = applicationFormRepository.save(applicationForm);
+
+        Optional.ofNullable(HelperUtils.getAdminSessionForAuthenticatedUser())
+                .ifPresentOrElse(adminSession -> {
+                    final SchemeEntity grantScheme = schemeRepository.findById(applicationForm.getGrantSchemeId())
+                            .orElseThrow(() -> new EntityNotFoundException("Could not find grant scheme with ID" + applicationForm.getGrantSchemeId()));
+
+                    grantScheme.setLastUpdated(Instant.now(clock));
```

Comment:
Makes it easier when accessing the static method in unit tests instead of having to use that horrible `MockedStatic` thing to assert against the values in the `Instant`